src/distances.jl

```diff
@@ -1,3 +1,5 @@
+import Statistics: mean
```

Comment:
```suggestion
using Statistics: mean
```
Unless you are adding methods to `mean`